optional_requirements.txt

```diff
@@ -0,0 +1 @@
+pywebview[gtk]
```

Comment:
I like the new optional requirements file. Please add it in setup.py too. See https://github.com/derrod/legendary/blob/cffb10188a11475f417cad394ceb929af44e6773/setup.py#L42 as example. Then you can execute pip install Rare[webview]